### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by rgcunha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# AngularJS Code Challenge
+
+## Description
+This is the boilerplate project for the AngularJS Code Challenge.
+
+## Dependencies
+* NodeJS (>=0.10.0)
```
Comment (erhangundogan) [inline README.md]: NodeJS 5.* would do better but not so strict

File: bower.json [hunk 2]
```diff
@@ -2,13 +2,7 @@
...
   "version": "0.0.0",
   "dependencies": {
-    "angular-animate": "~1.4.2",
-    "angular-cookies": "~1.4.2",
-    "angular-sanitize": "~1.4.2",
-    "angular-messages": "~1.4.2",
     "angular-route": "~1.4.2",
-    "moment": "~2.10.6",
-    "animate.css": "~3.4.0",
     "angular": "~1.4.2"
```
Comment (erhangundogan) [inline bower.json]: my bower complains about this
```bower angular extra-resolution Unnecessary resolution: angular#~1.4.2```

File: src/app/index.scss [hunk 3]
```diff
@@ -26,47 +26,9 @@ a {
...
-        line-height: 20px;
-      }
-      &.active a {
-        background-color: black;
-      }
-    }
-  }
-}
-
 .jumbotron {
```
Comment (erhangundogan) [inline src/app/index.scss]: this is bootstrap specific css

File: src/app/main/main.controller.spec.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    var MainController;
+
+    beforeEach(module('angularChallenge'));
+    beforeEach(inject(function(_$controller_) {
+      $controller = _$controller_;
+      MainController = $controller("MainController as vm", { $scope: $scope });
+    }));
+
+    it('should be registered', function() {
+      expect(MainController).not.toEqual(null);
```
Comment (erhangundogan) [inline src/app/main/main.controller.spec.js]: so test library will be jasmine?

File: bower.json (hunk 2 above)
Comment (ubenzer) [inline bower.json]: How about angular 1.5?

**Pull request closed without merging**